filters/exclusions.txt

```diff
@@ -442,5 +442,11 @@ $~object-subrequest
 ! End: EasyList Czech and Slovak
 !###########################
 !
+!### Peter Lowe's Blocklist ###
+! https://github.com/AdguardTeam/AdguardFilters/issues/175556
+||tags.tiqcdn.com^
```

Comment:
> Their filter dev is known not to unblock urls.

It's by design. I don't think that it should be fixed on our side. We are only responsible for AdGuard filters.

Reply:
Then all third-party exclusions would have to be removed.

Reply:
In other filters breakage is not acceptable.